pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/InTransformFunctionTest.java

```diff
@@ -51,6 +51,24 @@ public void testIntInTransformFunction() {
       assertEquals(intValues[i], inValues.contains(_intSVValues[i]) ? 1 : 0);
     }
   }
+  @Test
+  public void testIntNotInTransformFunction() {
```

Comment:
Can we add a e2e query execution test ? May be in one of the subclasses of `BaseQueriesTest` or `TransformQueriesTest` ?

Reply:
it's a bit hard b/c it will be rewrite as `NOT(IN(...))` as stated in the description.

Reply:
I see. I am ok with merging though